content/spin/cache.md

```diff
@@ -0,0 +1,66 @@
+title = "The Spin Registry Cache"
+template = "spin_main"
+date = "2022-03-14T00:22:56Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/cache.md"
+
+---
+
+- [The Spin Registry Cache](#the-spin-registry-cache)
+- [Clearing the Registry Cache](#clearing-the-registry-cache)
+  - [Viewing Registry Cache Files](#viewing-registry-cache-files)
+
+## The Spin Registry Cache
+
+A running Spin application can [fetch resources from remote registries](https://developer.fermyon.com/spin/spin-oci) and individual component sources via HTTP endpoints. These resources naturally consume network bandwidth. To ensure network efficiency, and to prevent waiting to download the same files every time an application is started, Spin automatically maintains a local `spin/registry` cache.
+
+## Clearing the Registry Cache
+
+Clearing the registry cache directory can be done by removing the `registry` directory entirely. The only direct effect of these actions is that Spin will have to pull again all component sources and static assets.
+
+### Viewing Registry Cache Files
+
+> **Note:** The registry cache contains no user-serviceable parts; the only safe way to make changes to the contents of the cache is through the Spin command line. The only operation a user can safely undertake is to delete the `registry` directory.
```

Comment:
I need to be more careful about using informal, metaphorical language like "user-serviceable parts" in my comments, in case you like them, and I have to ask myself "would a non-native speaker recognise the idiom here."

I think it's okay for now (partly because right now I can't think of another good concise way to say it!) but we might need to iterate on a more literal way of expressing this.